apps/full-site-editing/full-site-editing-plugin/readme.txt

```diff
@@ -1,5 +1,5 @@
 === Full Site Editing ===
-Contributors: alexislloyd, allancole, automattic, bartkalisz, codebykat, copons, dmsnell, get_dave, glendaviesnz, gwwar, iamtakashi, jeryj, Joen, jonsurrell, kwight, marekhrabe, mattwiebe, mkaz, mmtr86, mppfeiffer, noahtallen, nrqsnchz, nosolosw, obenland, okenobi, owolski, themeshaper, vindl
+Contributors: alexislloyd, allancole, automattic, bartkalisz, codebykat, copons, dmsnell, get_dave, glendaviesnz, gwwar, iamtakashi, jeryj, Joen, jonsurrell, kwight, marekhrabe, mattwiebe, mkaz, mmtr86, mppfeiffer, noahtallen, nrqsnchz, nosolosw, obenland, okenobi, owolski, iandstewart, vindl
```

Comment:
Now it's not in alphabetical order anymore. 😄 
https://gph.is/2fh0nG2

Reply:
But do we blame themeshaper or iandstewart for this?

Reply:
🤔 🤔 🤔

Reply:
Oops, I'll have to add myself to the list just to fix this up 😆